src/_guides/language/effective-dart/usage.md

```diff
@@ -721,22 +720,32 @@ then. That always happens after all initializers are run regardless of where
 useful. In fact, [dartdevc]({{site.webdev}}/tools/dartdevc)
 *requires* that it appear last.
 
-<div class="good">
+{% comment %}
+Question for Kathy: I've added a "Dart 2.0 note" below.
+Do we need to keep the last sentence above?
+{% endcomment %}
```

Comment:
Question for @kwalrath

Reply:
I like the note, and I think we should remove that sentence about dartdevc.

Reply:
done